### ISSUE TITLE
Feature Request: Copy/Paste Lighting Events

### ISSUE THREAD

**Issue by Fromen8923** [Ikeiwa/EditSaber#30]
Lights are arguably the most repetitive things in a chart. Would be great to be able to copy paste them like notes. Seems to me like it wouldn't be too difficult, as the lighting events are very similar to notes in the .json files.

**Comment by Tehelee** [comment]
Yeah this would be great to have, I ended up writing a script to pump out a pattern of blocks that I could edit into the json file. It'd be really great if he got multi-select and copy-paste working for all the events and walls.

**Comment by Ikeiwa** [comment]
I'm currently recoding the editor in a way that simplify the implementation of copy-pasting anything and undo/redo

**Comment by Fromen8923** [comment]
Awesome, thanks so much. Looking forward to the update.
